src/pages/users/new-user.tsx

```diff
@@ -79,7 +80,8 @@ export const getServerSideProps = withMultipleServerSideProps(
               userDetails: userCreateDetails,
               csrfToken,
               currentUser,
-              userGroups
+              userGroups,
+              currentUserVisibleForces: currentUser.visibleForces ?? ""
```

Comment:
I was thinking maybe we can define `currentUserVisibleForces` variable on the top and use it instead of repeating this line in every return. What do you think?

Reply:
yup, that is a good point, will make it easier to read